api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/QualityImprovement2020CohortQueries.java

```diff
@@ -5682,6 +5745,13 @@ public CohortDefinition getMQ13P4(Boolean den, Integer line) {
             hivMetadata.getYesConcept().getConceptId(),
             commonMetadata.getBreastfeeding().getConceptId());
 
+    CohortDefinition breastfeedingWithCargaViralHigherThan50 =
+        QualityImprovement2020Queries.getMQ13DenB5_PregnantCV50(
+            hivMetadata.getAdultoSeguimentoEncounterType().getEncounterTypeId(),
+            hivMetadata.getHivViralLoadConcept().getConceptId(),
+            hivMetadata.getYesConcept().getConceptId(),
+            commonMetadata.getBreastfeeding().getConceptId());
```

Comment:
Call the inner method directly here and pass the amount of vl copies

Reply:
Done